packages/instant-apps-components/src/components/instant-apps-app-guide/instant-apps-app-guide.tsx

```diff
@@ -1,18 +1,59 @@
-import { Component, Host, h } from '@stencil/core';
+import { Component, Host, h, Element } from '@stencil/core';
 
 @Component({
   tag: 'instant-apps-app-guide',
   styleUrl: 'instant-apps-app-guide.scss',
   shadow: true,
 })
 export class InstantAppsAppGuide {
+  @Element()
+  el: HTMLInstantAppsAppGuideElement;
 
   render() {
     return (
       <Host>
-        <slot>It Works!</slot>
+        <calcite-panel scale="s">
+          <span slot="header-content">App Guide <calcite-icon icon="lightbulb" scale="s"></calcite-icon></span>
```

Comment:
Is the header content going to be configurable? If so, it seems like this would be text that it can default to

`App Guide` will need to be translated so I think it'd good to move the string into the t9n files and load the messages through the `getMessages` util. 

Here's how's it's implemented in the other components:
https://github.com/Esri/instant-apps-components/blob/master/packages/instant-apps-components/src/components/instant-apps-social-share/instant-apps-social-share.tsx#L253

Reply:
The header of the component (App Guide) will not be configurable, though the title of the content area (in bold below the divider line) will be.  The app guide string has been added to the t9n file; I'll wire it up with `getMessages`.

Reply:
Is the text "App Guide" going to appear in the component header? If so then I think it needs to be configurable. I don't think users will always want this text visible?

Reply:
@kellyhutchins I'll chat with Design about this.  We hadn't talked about that header being configurable or optionally hidden.

Reply:
For now I've added a property that will allow the header to be hidden.  I've also wired up the translated strings to the header text.